components/bl-accordion-component/styles/index.less

```diff
@@ -80,7 +84,7 @@
       }
     }
 
-    &.active i {
+    &.active svg {
```

Comment:
Try to avoid using element selectors if it possible

`.collapse-icon | .collapse-icon-svg` - is more readable, isn't it?